node_modules/.package_versions.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
This file shouldn't exist. I think it's an artifact of mistakenly running `npm install` in the top-level directory or something like that?